src/main/scala/cromwell/engine/backend/BackendCall.scala

```diff
@@ -181,6 +182,22 @@ trait BackendCall {
       Future.successful(ExecutionHash("", None))
   }
 
+  def pollingInitialInterval: FiniteDuration
+  def pollingMaxInterval: FiniteDuration
+  def pollingMultiplier: Double
+  def pollingInitialGap: FiniteDuration
```

Comment:
Mokay - I think it's a bit confusing to default half of the parameters because they happen to be the same for the backends we currently have and let the other half undefined, but I won't make a fuss for this :smile: